sound/soc/sof/intel/hda.c

```diff
@@ -26,11 +26,15 @@
 #include <sound/intel-nhlt.h>
 #include <sound/sof.h>
 #include <sound/sof/xtensa.h>
+#include <trace/events/sof_intel.h>
 #include "../sof-audio.h"
 #include "../sof-pci-dev.h"
 #include "../ops.h"
 #include "hda.h"
 
+#define CREATE_TRACE_POINTS
+#include <trace/events/sof_intel.h>
```

Comment:
is line 29 required? Not sure why the same include file is referenced twice, in the second case *AFTER* the CREATE_TRACE_POINTS?

Reply:
Nope, just removed it. I didn't realized it was included already when I moved the `#define CREATE_TRACE_POINTS` in from `hda-trace`.